.flake8

```diff
@@ -11,7 +11,7 @@ exclude =
 max-complexity = 20
 max-line-length = 99
 # based on https://www.python.org/dev/peps/pep-0008/#maximum-line-length
-max-doc-length = 72
+max-doc-length = 80
```

Comment:
Can we add an internal ticket to bring it back later on? 🙏🏽

Reply:
Done